docs/New-Maintainer-Checklist.md

```diff
@@ -61,7 +61,7 @@ If they accept, follow a few steps to get them set up:
 - Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication)).
 - Ask them to disable SMS as a 2FA device or fallback on their GitHub account in favour of using one of the other authentication methods.
 - Ask them to (regularly) review remove any unneeded [GitHub personal access tokens](https://github.com/settings/tokens).
-- Add them to [Homebrew/brew's README](https://github.com/Homebrew/brew/edit/master/README.md).
+- Add them to Homebrew/brew's README and both man pages. See [this commit](https://github.com/Homebrew/brew/commit/8e092d0ae038f0af2a12c51d0fce58742c7516f3#diff-04c6e90faac2675aa89e2176d2eec7d8) for an example.
```

Comment:
Alternatively: add to the README and run (and commit the results of) `brew man`. That will cover all places that may be added in future.

Reply:
Nice, thanks, I wish I’d found that earlier. 😅 I’ve updated the words.